tests/Unit/CssInlinerTest.php

```diff
@@ -3215,6 +3216,7 @@ public function copyUninlinableCssToStyleNodeHasNoSideEffects()
         $expectedHtml = $subject->render();
 
         $styleElement = $domDocument->getElementsByTagName('style')->item(0);
+        self::assertInstanceOf(\DOMElement::class, $styleElement);
```

Comment:
It looks a bit messy having this here, but this is a messy test in the first place, so I'm fine with this.

Reply:
it's to cause the test to fail in the event that `DOMNodeList::item()` returns null.